android/app/src/main/kotlin/coop/polypoly/polypod/polyOut/PolyOut.kt

```diff
@@ -19,7 +19,8 @@ import kotlin.coroutines.EmptyCoroutineContext
 open class PolyOut(
     val context: Context
 ) {
-    private var readdirCache = mutableMapOf<String, Array<String>>()
+    private var readdirCache =
```

Comment:
Nit: You could rename this one to `readDirCache`, now that `readdir` is gone, but not super crucial, being a pretty local variable :D